packages/interface/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "cw-orch-polytone"
```

Comment:
Nit about the folder name... `interface` implies to me that it's an interface (i.e. the messages, responses, and general structs used by the polytone contracts. However, `cw-orch-polytone` is not that.

IMO, let's call the folder `cw-orch-polytone` or `deployer` or something like that.

A question for you for other libraries you might add something similar to (DAO DAO would be sick)... is this a package? Or perhaps it belongs in another folder. 

Keeping it in `packages` is fine for now, but please rename to `cw-orch-polytone` or `deployer` (slight preference for the former). Sorry in advance for this annoying nit pick. :joy:

Reply:
Will change the name, but that's a package ! That's something you can import from other projects to use the interface and associated state/artifacts.